portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -7097,6 +7130,44 @@ private boolean _isPasswordUnchanged(
 		return newEncPwd.equals(user.getPassword());
 	}
 
+	private void _onUserDoesNotExist(
+		String authType, String login, long companyId,
+		Map<String, String[]> headerMap) {
```

Comment:
Swap positions of `login` and `companyId` parameters so it matches `AuthPipeline.onMaxFailuresByEmailAddress` (etc.)